src/ui/templates/questions/questionsubmission.hbs

```diff
@@ -38,7 +38,7 @@
         {{#if errors.network}}
           <div class="yxt-QuestionSubmission-formRow">
             <div class="yxt-QuestionSubmission-formError" role="alert">
-              {{_config.networkErrorText}}
+              {{errors.network}}
```

Comment:
Because errors is plural here, it suggests to me that this is an array. I'm not sure if changing the QuestionSubmission model would have other effects though

Reply:
I think it's ok to keep it as `errors`, our validate function in questionsubmissioncomponent uses this field as well and it's possible we can more than one error store in there